### PR TITLE
[DUOS-1388][risk=no] Remove user.profileCompleted and user.properties.completed (Front end)

### PR DESCRIPTION
**Pull request by MiriamAlter**
- Removed all references to user.profileCompleted and user.properties.completed
- Implemented utils functions to calculate variable value when applicable
- Minor UI fixes for Admin Manage Users

Issue: https://broadworkbench.atlassian.net/browse/DUOS-1388

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/DataAccessRequestApplication.js [hunk 1]
```diff
@@ -1139,4 +1139,4 @@ class DataAccessRequestApplication extends Component {
   }
 }
 
-export default DataAccessRequestApplication;
+export default DataAccessRequestApplication;
```
Comment (rushtong) [inline src/pages/DataAccessRequestApplication.js]: minor nit - we don't need to remove this line break. In general, all files should end with a line break.

File: src/pages/ResearcherProfile.js [hunk 2]
```diff
@@ -231,7 +219,8 @@ export default function ResearcherProfile(props) {
   };
 
   const eraValidate = async () => {
-    // Temporary fix until eRACommons.js is updated to share re-render info.
+    // This function is currently unused but can check if a user's eRA Commons ID is in date.
+    // It will be used to update the researcherIncompleteFields component when eRACommons.js shares re-render info.
```
Comment (rushtong) [inline src/pages/ResearcherProfile.js]: If this is unused, it should be removed. Unused code triggers console errors.

**Review by rushtong — CHANGES_REQUESTED** [review]
See minor nits inline. Once the update profile functionality is fixed, please re-submit, thank you!

File: src/pages/ResearcherProfile.js (hunk 2 above)
Comment (MiriamAlter) [reply src/pages/ResearcherProfile.js]: It looks like it will have to be re-implemented when fixing the researcher profile page.

File: src/libs/ajax.js [hunk 3]
```diff
@@ -885,11 +885,10 @@ export const User = {
 
   update: async (user, userId) => {
     const url = `${await Config.getApiUrl()}/api/dacuser/${userId}`;
-    // We should not be updating the user's create date, associated institution, or library cards
+    // We should not be updating the user's create date or library cards
     let filteredUser = flow(
       cloneDeep,
       unset('updatedUser.createDate'),
-      unset('updatedUser.institution'),
```
Comment (rushtong) [inline src/libs/ajax.js:892]: I think this should still be unset. We need to update the user's `institutionId`, and not try to update the entire `institution` object itself.

**Comment by rushtong** [comment]
Thank you @MiriamAlter for all of your hard work on this! We can take this PR from here.

**Comment by rushtong** [comment]
Closing this PR for now and will incorporate the necessary changes in smaller PRs.
